### PR TITLE
update dependencies and work with cargo-fmt

### PR DESCRIPTION
**Pull request by DCjanus**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -27,20 +27,20 @@ travis-ci = {repository = "kbknapp/cargo-outdated"}
 name = "cargo-outdated"
 
 [dependencies]
-cargo = "0.35.0"
-docopt = "1.0.0"
-env_logger = "0.6.0"
+cargo = "0.36.0"
+docopt = "1.1.0"
+env_logger = "0.6.1"
```
Comment (Dylan-DPC) [inline Cargo.toml]:
```suggestion
env_logger = "0.6"
```

File: Cargo.toml [hunk 2]
```diff
@@ -27,20 +27,20 @@ travis-ci = {repository = "kbknapp/cargo-outdated"}
...
 semver = "0.9.0"
-serde = "1.0.11"
-serde_derive = "1.0.11"
-tabwriter = "~1.1.0"
-tempfile = "3"
-toml = "~0.5.0"
-failure = "0.1.1"
+serde = "1.0.92"
```
Comment (Dylan-DPC) [inline Cargo.toml]:
```suggestion
serde = "1"
```

File: Cargo.toml [hunk 3]
```diff
@@ -27,20 +27,20 @@ travis-ci = {repository = "kbknapp/cargo-outdated"}
...
+serde_derive = "1.0.92"
```
Comment (Dylan-DPC) [inline Cargo.toml]:
```suggestion
serde_derive = "1"
```

File: Cargo.toml [hunk 4]
```diff
@@ -27,20 +27,20 @@ travis-ci = {repository = "kbknapp/cargo-outdated"}
...
+tabwriter = "1.1.0"
```
Comment (Dylan-DPC) [inline Cargo.toml]:
```suggestion
tabwriter = "1"
```

File: Cargo.toml [hunk 5]
```diff
@@ -27,20 +27,20 @@ travis-ci = {repository = "kbknapp/cargo-outdated"}
...
+tempfile = "3.0.8"
```
Comment (Dylan-DPC) [inline Cargo.toml]: keep the original

File: Cargo.toml [hunk 6]
```diff
@@ -27,20 +27,20 @@ travis-ci = {repository = "kbknapp/cargo-outdated"}
...
+toml = "0.5.1"
```
Comment (Dylan-DPC) [inline Cargo.toml]:
```suggestion
toml = "0.5"
```

File: Cargo.toml [hunk 7]
```diff
@@ -27,20 +27,20 @@ travis-ci = {repository = "kbknapp/cargo-outdated"}
...
+failure = "0.1.5"
```
Comment (Dylan-DPC) [inline Cargo.toml]:
```suggestion
failure = "0.1"
```

File: Cargo.toml [hunk 8]
```diff
@@ -27,20 +27,20 @@ travis-ci = {repository = "kbknapp/cargo-outdated"}
...
 
 [dependencies.termcolor]
 optional = true
-version = "1.0"
+version = "1.0.5"
```
Comment (Dylan-DPC) [inline Cargo.toml]:
```suggestion
version = "1"
```

**Comment by Dylan-DPC** [comment]
Since crates follow semver, we don't need to specify the entire version and then have to update on every minor release.

**Comment by kornelski** [comment]
Are you sure about this? In the past I've found that using `= "1"` combined with `-Z minimal-versions` (that actually tests it) breaks A LOT of stuff.

**Pull request closed without merging**